cli/cmd/lql.go

```diff
@@ -94,5 +238,28 @@ func runLQLQuery(_ *cobra.Command, args []string) error {
 	if err := cli.OutputJSON(response); err != nil {
 		return errors.Wrap(err, "unable to format json response")
 	}
+
 	return nil
 }
+
+func runQuery(cmd *cobra.Command, args []string) error {
+	msg := "unable to run LQL query"
+	var response map[string]interface{}
+
+	query, err := inputQuery(cmd, args)
+	if err != nil {
+		return output(response, err, msg)
+	}
+
+	cli.Log.Debugw("running LQL query", "query", query)
+
+	if lqlCmdState.ValidateOnly {
+		// validate_only should compile
+		return CompileQueryAndOutput(query)
+	} else {
+		// !validate_only should should run
+		response, err = cli.LwApi.LQL.RunQuery(query, lqlCmdState.Start, lqlCmdState.End)
+	}
+
+	return output(response, err, msg)
```

Comment:
Here is an example of a more readable code, maybe you can try to replicate these modifications to other functions? If not, I can continue adding suggestions 😉

```suggestion
	query, err := inputQuery(cmd, args)
	if err != nil {
		return errors.Wrap(err, "unable to run LQL query")
	}

	cli.Log.Debugw("running LQL query", "query", query)

	if lqlCmdState.ValidateOnly {
		return CompileQueryAndOutput(query)
	}
	
	response, err := cli.LwApi.LQL.RunQuery(query, lqlCmdState.Start, lqlCmdState.End)
	if err != nil {
		return errors.Wrap(err, "unable to run LQL query")
	}

	if data, ok := response["data"]; ok {
		return cli.OutputJSON(data)
	}
	
	if err := cli.OutputJSON(response); err != nil {
		return errors.Wrap(err, "unable to format json response")
	}
	return nil
```

Reply:
Fixed.